features/steps/project/source/browse_files.rb

```diff
@@ -99,6 +112,22 @@ class Spinach::Features::ProjectSourceBrowseFiles < Spinach::FeatureSteps
     page.should have_content "Commit message"
   end
 
+  step 'I click on "Upload" in repo' do
+    click_button 'Upload'
+  end
+
+  step 'I click on "Upload file"' do
+    click_button 'Upload file'
+  end
+  
+  step 'I upload "user.feature"' do
+    attach_file(:file_upload, File.join('features', 'user.feature'))
+  end
+  
+  step 'I replace it with "user.feature"' do
+    attach_file(:file_upload, File.join('features', 'user.feature'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.